lsan_suppression.txt

```diff
@@ -0,0 +1,2 @@
+# Leak arises from OpenSSL p11 and SoftHSM2, where an engine is allocated during initilisation.
```

Comment:
It's not clear where it leaks. A trace would help.

Reply:
Makes sense. Will do!

Reply:
Done.